inference-engine/tools/compile_tool/main.cpp

```diff
@@ -96,31 +101,33 @@ DEFINE_string(iol, "", iol_message);
-    std::cout << "      -VPU_NUMBER_OF_SHAVES      <value>     "   << number_of_shaves_message     << std::endl;
-    std::cout << "      -VPU_NUMBER_OF_CMX_SLICES  <value>     "   << number_of_cmx_slices_message << std::endl;
-    std::cout << "      -VPU_TILING_CMX_LIMIT_KB   <value>     "   << tiling_cmx_limit_message     << std::endl;
-    std::cout                                                                                      << std::endl;
-    std::cout << " FPGA-specific options:                      "                                   << std::endl;
-    std::cout << "      -DLA_ARCH_NAME             <value>     "   << dla_arch_name                << std::endl;
+    std::cout                                                                                           << std::endl;
+    std::cout << " Common options:                             "                                        << std::endl;
+    std::cout << "    -h                                       "   << help_message                      << std::endl;
+    std::cout << "    -m                           <value>     "   << model_message                     << std::endl;
+    std::cout << "    -d                           <value>     "   << targetDeviceMessage               << std::endl;
+    std::cout << "    -o                           <value>     "   << output_message                    << std::endl;
+    std::cout << "    -c                           <value>     "   << config_message                    << std::endl;
+    std::cout << "    -ip                          <value>     "   << inputs_precision_message          << std::endl;
+    std::cout << "    -op                          <value>     "   << outputs_precision_message         << std::endl;
+    std::cout << "    -iop                        \"<value>\"    "   << iop_message                     << std::endl;
+    std::cout << "    -il                          <value>     "   << inputs_layout_message             << std::endl;
+    std::cout << "    -ol                          <value>     "   << outputs_layout_message            << std::endl;
+    std::cout << "    -iol                        \"<value>\"    "   << iol_message                     << std::endl;
+    std::cout                                                                                           << std::endl;
+    std::cout << " MYRIAD-specific options:                    "                                        << std::endl;
+    std::cout << "      -VPU_NUMBER_OF_SHAVES      <value>     "   << number_of_shaves_message          << std::endl;
+    std::cout << "      -VPU_NUMBER_OF_CMX_SLICES  <value>     "   << number_of_cmx_slices_message      << std::endl;
+    std::cout << "      -VPU_TILING_CMX_LIMIT_KB   <value>     "   << tiling_cmx_limit_message          << std::endl;
+    std::cout << "      -VPU_THROUGHPUT_STREAMS    <value>     "   << number_of_vpu_througput_streams   << std::endl;
```

Comment:
`VPU_THROUGHPUT_STREAMS` name is deprecated. Please, use `MYRIAD_THROUGHPUT_STREAMS`. And it would be great if you rename all myriad configs with `MYRIAD_` prefix.